gitlabbot/gitlabbot/util.go

```diff
@@ -84,9 +84,37 @@ func parseRepoInput(urlOrRepoPath string) (hostedURL string, repo string, err er
 	}
 
 	splitRepo := strings.Split(repo, "/")
-	if len(splitRepo) != 2 {
+	if !isValidArgs(splitRepo) {
 		return hostedURL, repo, fmt.Errorf("invalid arguments, expected `<owner/repo>`")
 	}
 
 	return hostedURL, repo, nil
 }
+
+func isValidArgs (args []string) bool {
+	if len(args) < 2 {
+		return false
+	}
+
+	for i, arg := range args {
+		if arg == "" {
+			return false
+		}
+
+		var regexPattern string
+		// check that owner is only alphanumeric
+		if i == 0 {
+			regexPattern = "^[a-zA-Z0-9_-]*$"
+		} else {
+			regexPattern = "^[a-zA-Z0-9_\\-\\.]*$"
```

Comment:
turns out i still need a slash for the dash unless it's at the end of the expression, and i needed to escape the escape '\'  lol

Reply:
you can use backticks ` instead of quotes "

Reply:
thanks!